cwms-data-api/src/test/java/cwms/cda/api/MeasurementControllerTestIT.java

```diff
@@ -0,0 +1,654 @@
+    private static final List<Stream> STREAMS_CREATED = new ArrayList<>();
+    private static final List<String> STREAM_LOC_IDS = new ArrayList<>();
+
+    @BeforeAll
+    public static void setup() throws SQLException {
+        String testLoc = "StreamLoc321"; // match the stream location name in the json file
+        createLocation(testLoc, true, OFFICE_ID, "STREAM_LOCATION");
+        STREAM_LOC_IDS.add(testLoc);
+        createAndStoreTestStream("ImOnThisStream2");
+    }
+
+    static void createAndStoreTestStream(String testLoc) throws SQLException {
+        createLocation(testLoc, true, OFFICE_ID, "STREAM");
+        CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+        db.connection(c -> {
+            StreamDao streamDao = new StreamDao(getDslContext(c, OFFICE_ID));
+            Stream streamToStore = new Stream.Builder()
+                    .withId(new CwmsId.Builder()
+                            .withOfficeId(OFFICE_ID)
+                            .withName(testLoc)
+                            .build())
+                    .withLength(100.0)
+                    .withLengthUnits("km")
+                    .build();
+            STREAMS_CREATED.add(streamToStore);
+            streamDao.storeStream(streamToStore, false);
+        }, CwmsDataApiSetupCallback.getWebUser());
+    }
+
+    @AfterAll
+    public static void tearDown() {
+        for (Stream stream : STREAMS_CREATED) {
+            try {
+                CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+                db.connection(c -> {
+                    StreamDao streamDao = new StreamDao(getDslContext(c, OFFICE_ID));
+                    try {
+                        streamDao.deleteStream(stream.getId().getOfficeId(), stream.getId().getName(), DeleteRule.DELETE_ALL);
+                    } catch (Exception e) {
+                        // ignore
+                    }
+                }, CwmsDataApiSetupCallback.getWebUser());
+            } catch (SQLException ex) {
+                throw new RuntimeException(ex);
+            }
+        }
+        STREAMS_CREATED.clear();
+        for(String measLoc: STREAM_LOC_IDS)
+        {
+            try {
+                CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+                db.connection(c -> {
+                    MeasurementDao measDao = new MeasurementDao(getDslContext(c, OFFICE_ID));
+                    try {
+                        measDao.deleteMeasurements(OFFICE_ID, measLoc, null, null, null, null, null, null, null, null, null, null, null);
+                    } catch (Exception e) {
+                        // ignore
+                    }
+                }, CwmsDataApiSetupCallback.getWebUser());
+            } catch (SQLException ex) {
+                throw new RuntimeException(ex);
+            }
+        }
+    }
+
+    @Test
+    @Disabled
```

Comment:
Why are all the tests disabled? Schema support?

Reply:
yeah, the tests will run against the prod schema and fail, since it doesn't have the measurement updates.

Reply:
Maybe consider adding a note to the disabled tag about that until the schema updates are merged in

Reply:
here's my idea: https://github.com/USACE/cwms-data-api/pull/917